ansible/playbook/tasks/update.yml

```diff
@@ -102,6 +102,11 @@
         name:
           - percona-qan-app # https://jira.percona.com/browse/PMM-6766
 
+    - name: Remove redundant ansible roles # https://jira.percona.com/browse/PMM-7827
```

Comment:
PMM-6766: PMM AMI authorized_keys to be removed
How task related to this change? Do we have keys in this role?